tests/hooks/test_lsp_symbols.py

```
@@ -0,0 +1,284 @@
+        assert is_code_symbol("a_b_c") is False  # length 5 < 9
+        assert is_code_symbol("one_two") is False  # only 1 underscore
+
+
+# ---------------------------------------------------------------------------
+# strip_zero_width
+# ---------------------------------------------------------------------------
+
+
+class TestStripZeroWidth:
+    def test_strips_zwsp(self):
+        assert strip_zero_width("foo​bar") == "foobar"
```

Ruff (0.15.14)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)